Rubberduck.Core/UI/Inspections/InspectionResultsControl.xaml

```diff
@@ -27,23 +27,16 @@
                 <Setter Property="Margin" Value="4" />
             </Style>
 
-            <CollectionViewSource x:Key="ResultsByInspectionType" Source="{Binding Results}">
-                <CollectionViewSource.GroupDescriptions>
-                    <PropertyGroupDescription PropertyName="Inspection" Converter="{StaticResource InspectionTypeConverter}" />
-                </CollectionViewSource.GroupDescriptions>
-            </CollectionViewSource>
-
-            <CollectionViewSource x:Key="ResultsByModule" Source="{Binding Results}">
-                <CollectionViewSource.GroupDescriptions>
-                    <PropertyGroupDescription PropertyName="QualifiedSelection.QualifiedName" />
-                </CollectionViewSource.GroupDescriptions>
-            </CollectionViewSource>
-
             <BitmapImage x:Key="CopyResultsImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/document-copy.png" />
             <BitmapImage x:Key="SettingsImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/gear.png" />
             <BitmapImage x:Key="RefreshImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/arrow-circle-double.png" />
             <BitmapImage x:Key="FixImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/tick.png" />
-            <BitmapImage x:Key="GroupByImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Custom/PNG/GroupBy.png" />
+
+            <BitmapImage x:Key="GroupByInspectionTypeImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/block.png" />
+            <BitmapImage x:Key="GroupByInspectionImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/light-bulb-code.png" />
+            <BitmapImage x:Key="GroupByLocationImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Custom/PNG/ObjectClass.png" />
+            <BitmapImage x:Key="GroupBySeverityImage" UriSource="pack://application:,,,/Rubberduck.Resources;component/Icons/Custom/PNG/Severity.png" />
```

Comment:
considering that the InspectionFilterToBooleanConverters are only scoped to the ToolBar where they're used, would it make sense to move the GroupBy and FilterBy images into that `Toolbar.Resources` section as well?

Reply:
Maybe.  I left them where they were so all the images would be in the same place more than any other reason.  The converters are in the `Toolbar.Resources` so I could see them when I was writing the xaml.